src/backend/adapters/record/base-record.ts

```diff
@@ -112,7 +112,6 @@ class BaseRecord {
    */
   async update(params): Promise<BaseRecord> {
     try {
-      this.storeParams(params)
```

Comment:
I'm not sure why it was called twice (before and after update). This function takes 100+ms for large records by itself event after refactoring. I decided to remove it because it seemed to work fine + `save` function `storeParams` only after update,